CITATIONS.md

```diff
@@ -36,53 +36,53 @@ If any GPU features of SUNDIALS were utilized, please also cite:
   author    = {Balos, Cody J and Gardner, David J and Woodward, Carol S and Reynolds, Daniel R},
   journal   = {Parallel Computing},
   publisher = {Elsevier},
-  volume    = {108},
-  pages     = {102836},
+  year   = {2022},
+  note   = {v5.3.0}
   year      = {2021},
   doi       = {10.1016/j.parco.2021.102836}
 }
 ```
 
 When using the ARKODE package from SUNDIALS, please also cite:
 
-```bibtex
-@article{reynolds2022arkode,
+  year   = {2022},
+  note   = {v6.3.0}
   title   = {{ARKODE: A flexible IVP solver infrastructure for one-step methods}},
   author  = {Reynolds, Daniel R and Gardner, David J and Woodward, Carol S and Chinomona, Rujeko},
   journal = {arXiv preprint arXiv:2205.14077},
   year    = {2022}
 }
 ```
 
-We also ask that users cite the documentation for the package and version that
-they are using rather than the combined SUNDIALS online guide:
+  year   = {2022},
+  note   = {v6.3.0}
 
 ```bibtex
 @Misc{arkodeDocumentation,
   author = {Daniel R. Reynolds and David J. Gardner and Carol S. Woodward and Cody J. Balos},
   title  = {User Documentation for ARKODE},
   year   = {2022},
   note   = {v5.2.0}
-}
-```
+  year   = {2022},
+  note   = {v6.3.0}
```

Comment:
Remove old year and note (should be v5.3.0). Looks like `updateVersion.sh` needs to be updated for `CITATIONS.md`.